dataactcore/read_config.py

```diff
@@ -0,0 +1,165 @@
+import os
+import os.path
+from os.path import expanduser, normpath, dirname, abspath
+import yaml
+import re
+
+VALUES_TO_LOG = {"CONFIG_BROKER":["path","use_aws","local",'aws_bucket', 'aws_role', 'aws_region', 'aws_create_temp_credentials',
```

Comment:
Good call on explicitly listing what to log. My first thought was "why not just use a short list of what _not_ to log" and then realized that's a security issue waiting to happen.